frontend/public/components/overview/index.tsx

```diff
@@ -764,30 +781,60 @@ class OverviewMainContent_ extends React.Component<OverviewMainContentProps, Ove
     const {statefulSets} = this.props;
     return _.map(statefulSets.data, (ss) => {
       const buildConfigs = this.getBuildConfigsForResource(ss);
-      const obj = {
-        ...ss,
-        kind: StatefulSetModel.kind,
-      };
-      const readiness = {
-        desired: ss.spec.replicas || 0,
-        ready: ss.status.replicas || 0,
-      };
       const pods = this.getPodsForResource(ss);
       const alerts = combinePodAlerts(pods);
       const services = this.getServicesForResource(ss);
       const routes = this.getRoutesForServices(services);
+      const obj = {
+        ...ss,
+        kind: StatefulSetModel.kind,
+      };
+      const status = <OverviewItemReadiness
+        desired={ss.spec.replicas}
+        href={`${resourceObjPath(obj, obj.kind)}/pods`}
+        ready={ss.status.replicas}
+      />;
+
       return {
         alerts,
         buildConfigs,
         obj,
         pods,
-        readiness,
         routes,
         services,
+        status,
       };
     });
   }
 
+  createPodItems(): OverviewItem[] {
+    const {pods} = this.props;
+    return _.reduce(pods.data, (acc, pod) => {
+      const owners = _.get(pod, 'metadata.ownerReferences');
+      if (_.isEmpty(owners)) {
```

Comment:
Prefer an early return.

```js
if (!_.isEmpty(owners)) {
  return acc;
}

// ...
```

Reply:
Do we want to exclude pods with a 'Succeeded' phase?

Reply:
I wonder if it shouldn't be a filter we add (but not for this PR).

I'd err on the side of hiding completed pods, succeeded or failed.